docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -181,7 +182,7 @@ To get metadata from the client and to set it on the server side, follow these s
 2. To get metadata on the client side, use the `GetMetadata` method of the [Tizen.Multimedia.Remoting.MediaController](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaController.html) class:
 
     ```csharp
-    var metadata = controller.GetMetadata();
+    var metadata = mediaController.GetMetadata();
     ```
 
 The following table lists all the media control metadata properties the client can request from the server.
```

Comment:
As we are using the word "following" in the sentence, it is recommended to use a colon (:) at the end of the sentence instead of a period (.).

The following table lists all the media control metadata properties the client can request from the server:

If this is not in your scope to change, please ignore my comments.

Reply:
Done